psp34_shiden/contracts/shiden34/src/lib.rs

```diff
@@ -15,9 +18,12 @@ pub mod shiden34 {
     use openbrush::{
         contracts::{
             ownable::*,
-            psp34::extensions::{
-                enumerable::*,
-                metadata::*,
+            psp34::{
+                extensions::{
+                    enumerable::*,
+                    metadata::*,
+                },
+                Internal,
```

Comment:
psp34::Internal is imported twice 
https://github.com/AstarNetwork/wasm-showcase-dapps/pull/27/files#diff-c084cdf45f42bdb6e563636cee056161da26af1be4bc7bde84451e27ad2836b4R140

Reply:
new PR will be created for this